__tests__/e2e/docker-compose.yml

```diff
@@ -44,13 +44,13 @@ services:
     networks:
       - elastic
   synthetic:
-    image: docker.elastic.co/experimental/synthetics:${SYNTHETICS_VERSION}-synthetics
```

Comment:
this will not test the current commit, I am changing it on https://github.com/elastic/synthetics/pull/193

Reply:
I just see that the code is mounted as volume.